R/methods.R

```diff
@@ -1,17 +1,85 @@
-#' Title
+#' Imputation & analysis Methods
+#'
+#' @description
+#'
+#' These functions determine what methods rbmi should use when creating
+#' the imputation models, generating imputed values and  pooling the results.
 #'
 #' @name method
-#' @param burn_in TODO
-#' @param burn_between TODO
-#' @param same_cov TODO
-#' @param n_samples TODO
-#' @param covariance TODO
-#' @param threshold TODO
-#' @param REML TODO
-#' @param type TODO
-#' @param verbose TODO
-#' @param seed TODO
-#' @return TODO
+#'
+#' @param burn_in a numeric that specifies how many observations should be discarded
+#' prior to extracting actual samples. Note that the sampler
+#' is initialised at the maximum likelihood estimates and an uninformative
```

Comment:
uninformative -> weakly informative (just because the prior for the covariance matrix is not really uninformative, and complete uninformativeness is usually very difficult to reach in practise.

Reply:
done